elephant/statistics.py

```diff
@@ -181,4 +181,56 @@ def fanofactor(spiketrains):
         fano = np.nan
     else:
         fano = spike_counts.var() / spike_counts.mean()
-    return fano
\ No newline at end of file
+    return fano
+
+def lv(v):
+    """
+    Calculate the measure of local variation LV for
+    a sequence of time intervals between events.
+
+    Given a vector v containing a sequence of intervals, the LV is
+    defined as:
+
+    .math $$ LV := \\frac{1}{N}\\sum_{i=1}^{N-1}
+
+                   \\frac{3(isi_i-isi_{i+1})^2}
+                          {(isi_i+isi_{i+1})^2} $$
+
+    The LV is typically computed as a substitute for the classical
+    coefficient of variation for sequences of events which include
+    some (relatively slow) rate fluctuation.  As with the CV, LV=1 for
+    a sequence of intervals generated by a Poisson process.
+
+    Parameters
+    ----------
+
+    v : quantity array, numpy array or list
+        Consecutive time intervals
+
+    Returns
+    -------
+    lvar : float or nan
+           The LV of the inter-spike interval of the input sequence.
+           If an empty list is specified, or if the sequence has less
+           than two one entries, F:=nan.
+
+    References
+    ----------
+    ..[1] Shinomoto, S., Shima, K., & Tanji, J. (2003). Differences in spiking
+    patterns among cortical neurons. Neural Computation, 15, 2823–2842.
+
+
+    """
+    # convert to array
+    v = np.array(v)
+
+    # ensure we have enough entries
+    if v.size<2:
+        return np.nan
+
+    # calculate LV
+    lvar = np.mean(
```

Comment:
A few things here:

You can avoid assigning the variable here by just doing `return np.mean(...`
`v[:-1] - v[1:]` is the same as `-1*np.diff(v)`, which simplifies to `np.diff(v)` due to squaring.
Mean is associative, so you can move the `3` outside the mean.
Further, due to power identities, you can move the division inside the power.
So this line would simplify to:
```Python
return 3.*np.mean(np.power(np.diff(v)/(v[:-1] + v[1:]), 2))
```
Which in my test, where `v=np.random.rand(10000)`, it reduced the calculation time by more than a half while giving the same result.